lib/auth/auth_with_roles.go

```diff
@@ -2395,6 +2395,7 @@ func (a *ServerWithRoles) UpsertAuthServer(ctx context.Context, s types.Server)
 	return a.authServer.UpsertAuthServer(ctx, s)
 }
 
+// Deprecated: Prefer paginated variant [ListAuthServers].
 func (a *ServerWithRoles) GetAuthServers() ([]types.Server, error) {
```

Comment:
Can we implement this by collecting all pages from ListAuthServers so there is only one set of logic?